business/apps.go

```diff
@@ -22,6 +22,14 @@ type AppService struct {
 	businessLayer *Layer
 }
 
+func joinMap(m1 map[string]string, m2 map[string]string) map[string]string {
```

Comment:
Not sure if this is right, a join should support multi values.
Example, map1 == version=v1; map2 == version=v2
After a join you would overwrite and have only version=v2.
This is going to cause perhaps some inconsistency and would miss real scenario, I think.

Example: I have a reviews app with version=v1, version=v2, version=v3, I'd like to filter by:
- version label is present (ok).
- version=v1 is present or version=v2 or version=v3 (all of these should send a valid app)

So, if the version=v1 is present in the service or any of the workload labels, if should belong to the app.

I know app are more complex because are not real entities, but a virtual group, but I think in that case would help.

Reply:
So, for the app then perhaps we may need that the join is something internally like
vesion=v1,v2,v3
As "," is not a valid character for a label, instead to create a map of arrays, it can be used a list of valid values for a label.
Then in the UI it can be used to filter in multivalue, just an idea.

Reply:
Omg @lucasponce I asked it to you and we talked about overwrite last value XD I'll change it

Reply:
Really ?
I remember I used the same example.
Well, next time I'll add some notes and an example in the issue to clarify :-)